### PR TITLE
added a plot to visualize performance

### PR DESCRIPTION
**Pull request by VRongve**

### CODE REVIEW

**Comment by HGodal** [comment]
I've fixed the merging issues, but wait with the merging until @bcwein has merged his pull request 👍🏼

File: Environment/DAT540EANN.yml [hunk 1]
```diff
@@ -8,3 +8,4 @@ dependencies:
   - pycodestyle
   - pydocstyle
   - scikit-learn
+  - matplotlib
```
Comment (bcwein) [inline Environment/DAT540EANN.yml:11]: 👍

**Review by bcwein — CHANGES_REQUESTED** [review]
Code seems good but it has conflicts with main branch. @HGodal Will fix these.

**Review by HGodal — APPROVED** [review]

**Review by bcwein — APPROVED** [review]
👍 😄

**Review by johannakinstad — APPROVED** [review]

**Review by ovejorgensen — APPROVED** [review]

**Pull request merged**